### PR TITLE
Style calendar item page

### PR DESCRIPTION
**Pull request by AliTopal89**
the new create page
screen shot 2017-07-16 at 10 54 41 pm

the new edit page 
screen shot 2017-07-16 at 10 53 59 pm

the new edit page with datepicker
screen shot 2017-07-16 at 10 54 13 pm

the new show page 
screen shot 2017-07-16 at 10 59 14 pm

the edit page was too close to the header therefore added `margin-top:30px` seems responsive

wanted the buttons to align nicely with the rest of the form made changes in custom.scss

### CODE REVIEW

**Comment by osondoar** [comment]
Thanks for the contribution!

I was trying to figure out why the calendar views were off and It looks like at some point we lost part of the original css for the calendar view in `custom.scss`, probably because of merge conflicts. I've restored some of that style as part of #82 . That PR fixed the `edit` view, but not the `new` view.

Can you rebase  your branch and resubmit the PR?

A few comments:
- The form in the view and edit pages can be defined just once in a partial (`_form.html.erb`). Take a look at the [terms views](https://github.com/rubyforgood/justice/tree/master/app/views/admin/terms) to see how you can reuse it the form across views. You should use the html in `edit.html.erb` and discard the form html in `new.html.erb`. 
That new partial can also contain the JS `Pikaday` definition, so we define it just once.

- With the new changes, using the `form-group` is no longer neccesary.

- The submit buttons currently say "Create Calendar" and "Update Calendar", can you rename them to "Save Event"?

File: app/views/admin/calendar_items/edit.html.erb [hunk 1]
```diff
@@ -1,16 +1,35 @@
+<script>
```
Comment (osondoar) [inline app/views/admin/calendar_items/edit.html.erb:1]: we can move this to a `_form.html.erb` partial

File: app/views/admin/calendar_items/edit.html.erb [hunk 2]
```diff
@@ -1,16 +1,35 @@
...
+       console.log(this.getMoment().format('Do MMMM YYYY'));
+      }
+    });
+  }
+</script>
 <article id="calendar-edit">
    <h1>Edit Event</h1>
    <dl id="meta-top">
 
      <%= form_for([:admin, @calendar_item]) do |f| %>
```
Comment (osondoar) [inline app/views/admin/calendar_items/edit.html.erb:22]: The form definition would also be part of the partial

File: app/views/admin/calendar_items/edit.html.erb [hunk 3]
```diff
@@ -1,16 +1,35 @@
...
+     <div class="form-group">
            <dt><%= f.label :title %></dt>
-           <dd><%= f.text_field :title %></dd>
+           <dd><%= f.text_field :title, class: "form-control" %></dd>
```
Comment (osondoar) [inline app/views/admin/calendar_items/edit.html.erb:25]: I believe that after you rebase, `form-control` is no longer neccesary

**Review by osondoar — CHANGES_REQUESTED** [review]

File: app/views/admin/calendar_items/edit.html.erb [hunk 4]
```diff
@@ -1,16 +1,35 @@
...
             <dt><%= f.label :body %></dt>
-            <dd><%= f.text_area :body %></dd>
+            <dd><%= f.text_area :body, class: "form-control" %></dd>
 
            <dt><%= f.label :start_time %></dt>
-           <dd id="lesson-date" name ="lesson-date"><%= f.text_field :start_time %></dd>
-       <%= f.submit "Update Calendar", class: " btn btn-primary", style: "margin-top: 10px" %>
+     </div>
+           <dd id="lesson-date" name ="lesson-date"><%= f.text_field :start_time, id: "datepicker", class: "form-control", type: "datetime"%></dd>
+       <%= f.submit "Update Calendar", class: " btn btn-primary", id: "custom-edit", style: "margin-top: 10px" %>
```
Comment (osondoar) [inline app/views/admin/calendar_items/edit.html.erb:33]: Let's rename "Update Calendar" to "Save Event"

File: app/views/admin/calendar_items/new.html.erb [hunk 5]
```diff
@@ -15,24 +15,28 @@
...
 
     <li>
       <%= f.label :start_time %>
-      <%= f.text_field :start_time, id: 'datepicker' %>
+      <%= f.text_field :start_time, id: 'datepicker',class: "form-control", type: "datetime" %>
     </li>
   </ul>
 
+
   <%= f.submit "Create Calendar", class: "btn btn-primary" %>
```
Comment (osondoar) [inline app/views/admin/calendar_items/new.html.erb:39]: Let's rename "Create Calendar" to "Save Event"

**Review by osondoar — CHANGES_REQUESTED** [review]

**Comment by AliTopal89** [comment]
Will look at these later today or tomorrow and I'll resubmit a pull request for sure. Thank you for your feedback. 👍

**Pull request closed without merging**